src/gamebryo/gamegamebryo.cpp

```diff
@@ -69,6 +69,11 @@ QDir GameGamebryo::dataDirectory() const
   return gameDirectory().absoluteFilePath("data");
 }
 
+QMap<QString, QDir> GameGamebryo::secondaryDataDirectories() const
```

Comment:
I'm not too happy with this solution. I'll probably expand on this in uibase pr.